src/bespokelabs/curator/prompter/prompter.py

```diff
@@ -104,9 +111,12 @@ def _completions(
         if self is None:
             raise ValueError("Prompter must be provided")
 
-        curator_cache_dir = os.environ.get(
-            "CURATOR_CACHE_DIR", os.path.expanduser("~/.cache/curator")
-        )
+        if working_dir is None:
+            curator_cache_dir = os.environ.get(
+                "CURATOR_CACHE_DIR", os.path.expanduser("~/.cache/curator")
```

Comment:
~/.cache/curator should be a global variable at the top of the file.

_CURATOR_DEFAULT_CACHE_DIR = "~/.cache/curator"